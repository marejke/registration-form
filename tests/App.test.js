import { h } from 'preact';
import RegistrationForm from '../components/form';
import render from 'preact-render-to-string';

test('Tests for App to have been rendered', () => {
	const tree = render(<RegistrationForm />);
	expect(tree).toMatchSnapshot();
});
