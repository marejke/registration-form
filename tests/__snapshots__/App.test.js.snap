// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for App to have been rendered 1`] = `"<div><undefined></undefined></div>"`;
